tests/tools/cohort_tracking/test_cohort_tracking.py

```diff
@@ -0,0 +1,169 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import ehrapy as ep
+import ehrapy.tools.feature_ranking._rank_features_groups as _utils
+from ehrapy.io._read import read_csv
+
+CURRENT_DIR = Path(__file__).parent
+_TEST_DATA_PATH = f"{CURRENT_DIR.parent}/test_data_features_ranking"
+
+
+def _compare_dict_equal(dict1, dict2, tolerance=1e-9):
```

Comment:
Such bloat can be moved to the `conftest` file maybe?

Reply:
even better got rid of that entirely by moving away from the nested dict thing